integrasjon/dokgen-klient/src/main/java/no/nav/foreldrepenger/melding/integrasjon/dokgen/dto/FellesDokumentdata.java

```diff
@@ -1,18 +1,20 @@
 package no.nav.foreldrepenger.melding.integrasjon.dokgen.dto;
 
-import java.time.LocalDate;
-
 public class FellesDokumentdata {
     private String søkerNavn;
     private String søkerPersonnummer;
     private String fritekst;
-    private LocalDate brevDato;
+    private String brevDato;
+    private boolean erAutomatiskBehandlet;
+    private String erKopi;
```

Comment:
Hvorfor String? Bør vel være boolean sånn som den over? Da kan du bruke if i stedet for "eq JA/NEI" i handlebars på andre siden også?

Reply:
Ja, enig. Det er verdier som kommer fra basen, men bedre å sende over som boolean..